src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Win11/Styles/ComboBox.xaml

```diff
@@ -270,9 +270,6 @@
                                         BorderThickness="1"
                                         CornerRadius="{DynamicResource PopupCornerRadius}"
                                         SnapsToDevicePixels="True">
-                                        <Border.RenderTransform>
```

Comment:
This is used for animation and we need to evaluate the effect of removing this properly. We should leave it here for now and get back to this issue when we are doing performance evaluation for dynamic resources.